<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rapprochement</class>
 <widget class="QDialog" name="rapprochement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>660</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rapprochement</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/logo.png</normaloff>../ressources/logo.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="titre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rapprochement</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formulaireLayout">
    <property name="horizontalSpacing">
     <number>7</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dateDateEdit">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="heureLabel">
      <property name="text">
       <string>Heure</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTimeEdit" name="heureTimeEdit">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="referenceLabel">
      <property name="text">
       <string>Référence</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="referenceLineEdit">
      <property name="placeholderText">
       <string>R1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="memoLabel">
      <property name="text">
       <string>Mot explicatif</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="memoLineEdit">
      <property name="placeholderText">
       <string>Achat</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="operationLabel">
      <property name="text">
       <string>Opération</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QGridLayout" name="grilleOperationLayout">
      <item row="0" column="3">
       <widget class="QLabel" name="labelTitreVide">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTitreCompte">
        <property name="text">
         <string>Compte</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelTitreDebit">
        <property name="text">
         <string>Crédit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTitreCredit">
        <property name="text">
         <string>Débit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="ajouterOperationPushButton">
      <property name="text">
       <string>Ajouter une Opération</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="ModifierPushButton">
      <property name="text">
       <string>Modifier la Transaction</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>531</width>
     <height>173</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formulaireCompteLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelCompte">
      <property name="text">
       <string>Compte</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ValiderButton">
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="QuitterButton1">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelSoldeRapprochement">
      <property name="text">
       <string>Solde dernier rapprochement</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelSoldeRapprochementValeur">
      <property name="text">
       <string>€</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelSoldeActuel">
      <property name="text">
       <string>Solde actuel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="labelSoldeActuelValeur">
      <property name="text">
       <string>€</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelTransaction">
      <property name="text">
       <string>Transaction</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="listeTransaction"/>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="listeCompte"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ChangementPushButton">
      <property name="text">
       <string>Selectionner un autre compte</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="TerminerPushButton">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>630</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sauver et Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ValiderPushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>630</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider le compte</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
